app/assets/stylesheets/application.scss

```diff
@@ -10,4 +10,15 @@
 @import 'foundations';
 @import 'nav';
 @import 'root_path';
+
+
+.box {
+  background-color: $taupe;
+  display: inline-block;
+  margin: 5px;
+  padding: 5px;
+}
+
+
```

Comment:
@JonathanSR can you take this out? I added it to the foundations stylesheet. Thanks!